src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -24,12 +24,20 @@
 
 #include <toolkits/drawing_classifier/data_preparation.hpp>
 #include <toolkits/drawing_classifier/drawing_classifier.hpp>
+#include <core/logging/assertions.hpp>
+#include <toolkits/coreml_export/mlmodel_include.hpp>
+#include <core/util/string_util.hpp>
```

Comment:
Let's keep these alphabetized

Reply:
Sure thing!

Reply:
This is the debug relevant headers that I used to print out the protobuf messages. I will trim them.